decayers/ERElasticScattering.cxx

```diff
@@ -215,30 +240,31 @@ Bool_t ERElasticScattering::Stepping() {
 }
 
 void  ERElasticScattering::ThetaRangesLab2CM(Double_t pM, Double_t tM) {
+  LOG(DEBUG) << "ERElasticScattering::ThetaRangesLab2CM(pM = " << pM << ", tM = " << tM << ")" << FairLogger::endl;
   Double_t rAng = fThetaRangeCenter*DegToRad();
   Double_t ratio = pM/tM;
   Double_t ratio2 = ratio*ratio;
-  Double_t rdTheta = fThetaRangedTheta*TMath::DegToRad(); // Detectors rdTheta
+  Double_t rdTheta = fThetaRangedTheta*DegToRad(); // Detectors rdTheta
   if (fRegisterIonStatus == kPROJECTILE) {
     // Projectile Ion
     if (pM != tM) {
-      fThetaMin = TMath::RadToDeg()*acos( -ratio*sin(rAng-rdTheta)*sin(rAng-rdTheta)
+      fThetaMin = RadToDeg()*acos( -ratio*sin(rAng-rdTheta)*sin(rAng-rdTheta)
                 + cos(rAng-rdTheta)*sqrt(1.-ratio2*sin(rAng-rdTheta)*sin(rAng-rdTheta)) );
-      fThetaMax = TMath::RadToDeg()*acos( -ratio*sin(rAng+rdTheta)*sin(rAng+rdTheta)
+      fThetaMax = RadToDeg()*acos( -ratio*sin(rAng+rdTheta)*sin(rAng+rdTheta)
                 + cos(rAng+rdTheta)*sqrt(1.-ratio2*sin(rAng+rdTheta)*sin(rAng+rdTheta)) );
     }
     else {
-      fThetaMin = TMath::RadToDeg()*(2.*rAng - rdTheta);
-      fThetaMax = TMath::RadToDeg()*(2.*rAng + rdTheta);
+      fThetaMin = RadToDeg()*(2.*rAng - rdTheta);
+      fThetaMax = RadToDeg()*(2.*rAng + rdTheta);
     }
 
     LOG(DEBUG) << "  N15: CMTheta1: " << fThetaMin << ", CMTheta2: " << fThetaMax
```

Comment:
ERElasticScattering is a general class for reaction 2in2. Please do not specify it here

Reply:
Ok! I fixed It. I removed "N15" and wrote "Projectile".